### PR TITLE
created Modal to confirm goal deletion

### PR DESCRIPTION
**Pull request by mowfaqqa**
- Created modal to confirm if a goal should be deleted
- Created button labeled "delete Goal" to display modal component
- fixes issue #402 
- Live link https://goals.zuri.chat/  
NB: Click on delete Goal to display modal

### CODE REVIEW

**Comment by KelvinAkpobome** [comment]
Please resolve conflict

File: client/src/components/Deletemodal/Deletemodal.js [hunk 1]
```diff
@@ -1,3 +1,45 @@
...
+            <button className="cancel-modal" onClick={exitModal}>X</button>
+            <div className="image">
+               <img src={deleteDataImg.deleteImg} className="d-img1" alt="view" />
+               <img src={deleteDataImg.deleteimg2} className="d-img2" alt="view" />
+               <img src={deleteDataImg.deleteimg3} className="d-img3" alt="view" />
+               <img src={deleteDataImg.deleteimg3} className="d-img3-2" alt="view" />
+               <img src={deleteDataImg.deleteimg4} className="d-img4" alt="view" />
+               <img src={deleteDataImg.deleteimg5} className="d-img5" alt="view" />
+               <img src={deleteDataImg.deleteimg6} className="d-img6" alt="view" />
+               <img src={deleteDataImg.starimg} className="star-img" alt="view" />
```
Comment (Asylcreek) [inline client/src/components/Deletemodal/Deletemodal.js:26]: Please refactor this to use a map function, as it seems like the only thing changing is the classname.

So something like
```jsx
{[{image: 'deleteimg6', className: 'd-img6'}, {image: 'deleteimg2', className: 'd-img2'}].map((el) => <img src={deleteDataImg[el.image]} className={el.className} alt="view" />
```

If you are even feeling very excited, you can put all the possible options into a variable outside of the function and just loop through that variable instead.

Something like this
```jsx
const deleteImagesData = [{image: 'deleteimg6', className: 'd-img6'}, {image: 'deleteimg2', className: 'd-img2'}];

const DeleteModal = () => {
```

Then 
```jsx
{deleteImagesData.map((el) => <img src={deleteDataImg[el.image]} className={el.className} alt="view" />}
```

File: client/src/components/Deletemodal/Deletemodal.js [hunk 2]
```diff
@@ -1,3 +1,45 @@
+
+import React from 'react'
```
Comment (Asylcreek) [inline client/src/components/Deletemodal/Deletemodal.js:2]: Does the linter complain if React isn't imported?

**Review by Asylcreek — CHANGES_REQUESTED** [review]

File: client/src/components/Deletemodal/Deletemodal.js (hunk 2 above)
Comment (mowfaqqa) [reply client/src/components/Deletemodal/Deletemodal.js:2]: It doesn't complain

File: client/src/components/Deletemodal/Deletemodal.js (hunk 1 above)
Comment (mowfaqqa) [reply client/src/components/Deletemodal/Deletemodal.js:26]: Let me get to it then..thanks

File: client/src/components/Deletemodal/Deletemodal.js (hunk 2 above)
Comment (Asylcreek) [reply client/src/components/Deletemodal/Deletemodal.js:2]: Then please remove all React imports

Comment (Asylcreek) [reply client/src/components/Deletemodal/Deletemodal.js:1]: The react import is still here

**Review by Asylcreek — CHANGES_REQUESTED** [review]

**Comment by mowfaqqa** [comment]
I've removed the react syntax chief

File: client/src/index.css [hunk 3]
```diff
@@ -21,3 +21,166 @@ body {
...
+  overflow-x: hidden;
+  z-index: 1;
+}
+.delete-modal {
+  width: 485px;
+  height: 508px;
+  margin-top: 5rem;
+  background: #ffffff;
+  border-radius: 5px;
+  z-index: 1;
```
Comment (Asylcreek) [inline client/src/index.css:43]: Please move all these styles into a DeleteModal.styled.js file. and write the styles with styled-components

**Review by Asylcreek — CHANGES_REQUESTED** [review]

**Pull request merged**